src/extension/background.js

```diff
@@ -330,7 +330,7 @@ function toggle(id) {
 function getHelpLanguage() {
   const uLang = chrome.i18n.getUILanguage();
   const lang = LANGS.find((l) => uLang.replace('-', '_') === l
-    || l.startsWith(uLang.split('_')[0])) || LANGS[0];
+    || l.startsWith(uLang.split('-')[0])) || LANGS[0];
```

Comment:
Hmm, because this worked fine before for other languages, and we don't have control over locale formats in different operating systems and browsers, I'd feel safer if we used a regexp with both `-` and `_` here:
```suggestion
    || l.startsWith(uLang.split(/[-_]/)[0])) || LANGS[0];
```

Reply:
Sure! Makes sense 👍🏻